report_account_move_settlement/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+- This module adds custom invoice with settlement reports to account_move
```

Comment:
```suggestion
- Adds settlement report print function to account.move

The added function is intended to be used by auction sellers so that they can keep
the details of the settlement invoice for their record.
```